solr/core/src/java/org/apache/solr/core/SchemaCodecFactory.java

```diff
@@ -151,7 +151,7 @@ public Codec getCodec() {
   }
 
   /**
-   * This class exists because Lucene95HnswVectorsFormat's getMaxDimensions method is final and we
+   * This class exists because Lucene99HnswVectorsFormat's getMaxDimensions method is final and we
```

Comment:
why not directly reference it using `{@link ...}`.  It's take less chars overall!  (no need to refer to "method")